### ISSUE TITLE
`wl` script support.

### ISSUE THREAD

**Issue by hongyi-zhao** [njpipeorgan/wolfram-language-notebook#31]
I'm very glad to know this wonderful vscode plugin [here](https://community.wolfram.com/groups/-/m/t/2416983), but I've the following question:

Can this plugin support writing `wl` script, i.e., the file with `.wl` extension, directly in vscode? I've noticed that the official [`vscode-wolfram`](https://github.com/WolframResearch/vscode-wolfram) plugin has this feature.

Regards,
HZ

**Comment by wuyudi** [comment]
you could check the https://github.com/kenkangxgwe/lsp-wl/tree/develop.

**Comment by hongyi-zhao** [comment]
Do you mean the following ugly formatted style?
![image](https://user-images.githubusercontent.com/11155854/176329635-7485bdd1-75b4-4eb7-990c-564854587215.png)

**Comment by wuyudi** [comment]
@hongyi-zhao  no, it's just a lsp. You could run script with something like code-runner. https://web.archive.org/web/20220418065125/https://gitee.com/asdasd1dsadsa/ZhiHu-asdasd1dsadsa/blob/master/Article/%E4%B8%BAWolframEngine%E9%85%8D%E7%BD%AEVSCode%EF%BC%88Windows%EF%BC%89.md
gitee sucks.

**Comment by hongyi-zhao** [comment]
Do you mean the output shown  in the following panel:

![image](https://user-images.githubusercontent.com/11155854/176566846-1abb6e5c-4ac2-40c0-a397-a41f612c9156.png)

Anyway, it seems that this feature is far cry from notebook's Inline evaluation mode.

**Comment by wuyudi** [comment]
y, I mean that.

from py's view: it's just `.py` vs `.ipynb`

lsp. -> `.py`
Inline evaluation mode. -> `.ipynb`
